app/src/main/resources/com/oracle/javafx/scenebuilder/app/menubar/MenuBar.fxml

```diff
@@ -300,6 +300,20 @@
         <Menu fx:id="helpMenu" mnemonicParsing="false" text="%menu.title.help">
           <items>
             <MenuItem fx:id="helpMenuItem" mnemonicParsing="true" text="%menu.title.scene.builder.help" />
+                  <SeparatorMenuItem mnemonicParsing="false" />
+                  <Menu mnemonicParsing="false" text="%menu.title.help.javafx">
+                     <items>
+                        <MenuItem fx:id="gettingStartedMenuItem" mnemonicParsing="true" text="%menu.title.help.openjfx.getting.started" />
+                        <MenuItem fx:id="apiDocMenuItem" mnemonicParsing="true" text="%menu.title.help.openjfx.api.docs" />
+                        <MenuItem fx:id="cssReferenceGuideMenuItem" mnemonicParsing="true" text="%menu.title.help.openjfx.css.reference" />
+                        <MenuItem fx:id="fxmlIntroductionMenuItem" mnemonicParsing="true" text="%menu.title.help.openjfx.fxml.reference" />
+      				  <MenuItem fx:id="gettingStartedWithKotlinMenuItem" mnemonicParsing="true" text="%menu.title.help.openjfx.getting.started.kotlin" />
```

Comment:
Replace TABS with SPACES

Reply:
I missed this one, it's fixed now. Well, I have to play with checkstyle to include FXMLs as well.